redhat/overlays/Dockerfile.netcat

```diff
@@ -1,3 +1,3 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:9c810eccada0cc6f25b46eeeea4516bab01828faef4fdd2fe49c760518b90dd1
 
-RUN microdnf -y install nc
+COPY netcat /usr/bin
```

Comment:
We need to think about this and perhaps consult with prodsec. I think having an executable of unknown provenance in the image might not work due to security concerns.

Reply:
That makes sense. We can just forgo the hermetic build on this image then for the time being just so we can have an built image?

Reply:
Yeah - let's do that.